src/Adapter/Presenter/Module/ModulePresenter.php

```diff
@@ -67,7 +67,7 @@ public function present($module)
         $attributes['starsRate'] = str_replace('.', '', round($attributes['avgRate'] * 2) / 2); // Round to the nearest 0.5
 
         $moduleInstance = $module->getInstance();
-        if ($moduleInstance !== null && $moduleInstance->getMultistoreCompatibility() !== null) {
+        if ($moduleInstance !== null) {
```

Comment:
It's best to use duck typing here:
```suggestion
        if ($moduleInstance instanceof Module) {
```